### ISSUE TITLE
Removing dead Google web service-specific UI

### ISSUE THREAD

**Issue by Eloston** [Eloston/ungoogled-chromium#1131]
This is a meta-issue that tracks issues regarding the removal of unused/dead UI as a result of removing Google web service integration.

Currently: #224, #369, #999, #1130

**Comment by Ahrotahn** [comment]
I've started working on this and have a patch that resolves #224, #369, and #1130 in a simple manner.  

After reading through #999 I did some experimentaion.  One of the simplist solutions would be to remove these two patches:
`core/inox-patchset/0014-disable-translation-lang-fetch.patch`
`core/ungoogled-chromium/disable-translate.patch`

Everything then works the same as it did previously due to the URL substitution, but now UC can be launched with the `--translate-script-url` and `--translate-security-origin` flags to allow the translation feature to function again.  The "Translate to" context menu item can be set to only show if the script url has been set.  

Because those flags are set by default in the bad_flags_prompt file there will be a dismissable notice on startup saying:
> "You are using an unsupported command-line flag.  \<flag\>.  Stability and security will suffer."

I feel like that is a good compromise considering it's possible to send PII to whatever entity you use for your translations, but would like to hear other opinions on handling it that way.

**Comment by Ahrotahn** [comment]
I forgot to mention that my reasoning for handling the translation changes that way is due to maintenance concerns.  The flags and the notice are all already in the chromium source, so there is nothing special that needs to be done on our end.  Only a few lines needed to be added to remove the translation UI.  

If that gets accepted, then it might be a good idea to mention on the wiki that the translation funcationality can be resored to the Google default by setting the flags to:
```
--translate-script-url="https://translate.googleapis.com/translate_a/element.js"
--translate-security-origin="https://translate.googleapis.com/"
```

I'd be curious if there are other translation service URLs that would work as replacements for Google's.
